examples/web/readme.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment,@typescript-eslint/ban-ts-comment */
+import {CacheGet, CacheClient, Configurations, CredentialProvider} from '@gomomento/sdk-web';
+// @ts-ignore
+// This global is required in order to use the Web SDK outside of a browser
+global.XMLHttpRequest = require('xhr2');
```

Comment:
Could we init this if undefined down in Constructor? Not for this pr just curious if could hide this from users.

Reply:
if we hide it from them and they copy the code and try to run it then it won't run.  So I am torn between making it succinct vs. not surprising someone with weird errors if they copy/paste.  Open to feedback.

Reply:
if we hid it from them though couldn't we just set the `global.XMLHttpRequest` underhood if needed so no errors?

Reply:
we don't want the SDK to have a dependency on that xhr2 library, which is not necessary in a browser (and probably wouldn't even work there)

Reply:
ahh ok I see. and there no way we could lazy require it underhood so lib only got loaded if needed? Or just don't even want to have it in package.json at all cause could cause conflicts and adds to package size?

Reply:
we could make some example utils file with a helper function called `initializeWebSdkForNodeEnvironment` or something.  but that line would still need to be called in whatever examples we include here that are intended to be run on node.

tbh the examples here should probably be migrated over to be 100% browser examples.  Like a little web app you launch that serves up some simple pages that load in your browser.  But I don't have the expertise for that.

Reply:
we have it in the examples package.json but i don't want it in the SDK package.json, correct.

Reply:
that makes sense to me just try to have demos always running in browser for user and then just this advanced section in docs somewhere if trying to use websdk from nodejs process.

Reply:
filed a ticket about creating browser-based examples for web sdk